testsuite/fuzzer/fuzz/Cargo.toml

```diff
@@ -21,7 +21,7 @@ aptos-vm-environment = { workspace = true }
 arbitrary = { workspace = true, features = ["derive"] }
 base64 = { workspace = true }
 bcs = { workspace = true }
-libfuzzer-sys = "0.4"
+libfuzzer-sys = "0.4.9"
```

Comment:
Should this be a workspace as well and the actual version in main Cargo.toml?

Reply:
It's a local dependency. I don't think we need it at workspace level.